src/toolkits/drawing_classifier/drawing_classifier.cpp

```diff
@@ -24,12 +24,20 @@
 
 #include <toolkits/drawing_classifier/data_preparation.hpp>
 #include <toolkits/drawing_classifier/drawing_classifier.hpp>
+#include <core/logging/assertions.hpp>
+#include <toolkits/coreml_export/mlmodel_include.hpp>
+#include <core/util/string_util.hpp>
 
 namespace turi {
 namespace drawing_classifier {
 
 namespace {
 
+using CoreML::Specification::NeuralNetworkLayer;
+using CoreML::Specification::NeuralNetworkPreprocessing;
+using CoreML::Specification::SizeRange;
+using turi::coreml::MLModelWrapper;
```

Comment:
Let's keep these organized with the below. Line 39 here is fully redundant with 41

Reply:
yes. I used those for debugging. Thanks for pointing it out. I will do clean up.